cbcbeat/splittingsolver.py

```diff
@@ -75,6 +79,25 @@
     pass
 
 
+class Merger:
+    def __init__(self, ode_space, vur_space):
```

Comment:
Typing?
```suggestion
    def __init__(self, ode_space: dolfin.FunctionSpace, vur_space:dolfin.FunctionSpace):
```
(if dolfin is imported, I don't think it is, so we would need to add that as well

Reply:
I think `cbcbeat` still uses the classical `from dolfin import *` syntax so we actually don't need `dolfin` :p